query/src/test/java/org/infinispan/query/blackbox/IndexingWithPersistenceTest.java

```diff
@@ -7,11 +7,9 @@
 import static org.testng.AssertJUnit.assertNull;
 import static org.testng.AssertJUnit.assertTrue;
 
-import java.util.ArrayList;
-import java.util.Arrays;
 import java.util.Collections;
-import java.util.Comparator;
 import java.util.List;
+import java.util.Set;
```

Comment:
The changes in this class seem unrelated to the issue.

Reply:
Unrelated cleanup that snuck in, Reverted.